tasks/translation/models.py

```
@@ -0,0 +1,40 @@
+from datetime import datetime
+from typing import Dict
+from cognee.core.datapoint import DataPoint
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+
+@dataclass
+class TranslatedContent(DataPoint):
```

Pylint (3.3.7)

[refactor] Too many instance attributes (9/7)

(R0902)